src/pypromice/process/L1toL2.py

```diff
@@ -141,6 +156,248 @@ def toL2(L1, T_0=273.15, ews=1013.246, ei0=6.1071, eps_overcast=1.,
                                              T_0, T_100, ews, ei0)                   
     return ds
 
+def differenceQC(ds):
```

Comment:
What is `differenceQC`? Just glancing at the code, this appears to be a persistence check? That is, checking if a value has gotten stuck in a persisting state for a certain period of time. If so, I would rename to `persistenceQC`.

Reply:
Yep, it is a persistence/static check. We will rename it to staticQC.